energy/energy.ino

```diff
@@ -45,11 +49,13 @@ void setup()
 #endif
 
     // Set the product UID and set the sync mode to be "minimum"
-    J *req = notecard.newRequest("hub.set");
-    JAddStringToObject(req, "product", myProductID);
-    JAddStringToObject(req, "mode", "minimum");
-    notecard.sendRequest(req);
-
+    J *req;
+    if (myProductID[0]) {
+      req = notecard.newRequest("hub.set");
+      JAddStringToObject(req, "product", myProductID);
+      JAddStringToObject(req, "mode", "minimum");
```

Comment:
Looks like `"mode": "minimum"` should be offered to `hub.set`, regardless of `"product"` being set.

Reply:
:+1: My first thought was that we'd leave it alone so not to surprise the developer, in the same way we don't want to mess with the `PRODUCT_UID`. But given this is an energy example, minimum is probably the better choice. I'll also update the readme so the developer is aware of the mode change.